routing/dht/routing.go

```diff
@@ -84,7 +84,10 @@ func (dht *IpfsDHT) GetValue(ctx context.Context, key key.Key) ([]byte, error) {
 	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
 
-	vals, err := dht.GetValues(ctx, key, 16)
+	// retrieve a majority of the expected record count
+	majority := (KValue / 2) + 1
```

Comment:
> 20 and 11.

No, `KValue = 10`. `(KValue / 2) + 1 = 6`.

we maybe should use `KValue` for this, but it would have to be the whole `KValue` and `KValue` should then increase (likely not what we want). we can instead make another parameter:

```
const RecordLookupThreshold = 16
```

---

Also, this comment is incorrect.
```
// K is the maximum number of requests to perform before returning failure.
```
is incorrect. if we want the `k` from kademlia, then it _should_ mean:

```
// K is the number of closest peers to find
```
and it looks like we're mostly using it this way here:

```
jbenet @ lorien : ~/go/src/github.com/ipfs/go-ipfs/routing/dht * master > grep KValue *
handlers.go:var CloserPeerCount = KValue
lookup.go:	tablepeers := dht.routingTable.NearestPeers(kb.ConvertKey(key), KValue)
lookup.go:	out := make(chan peer.ID, KValue)
lookup.go:	peerset := pset.NewLimited(KValue)
grep: pb: Is a directory
routing.go:	rtp := dht.routingTable.NearestPeers(kb.ConvertKey(key), KValue)
routing.go:	for p := range dht.FindProvidersAsync(ctx, key, KValue) {
routing.go:	peers := dht.routingTable.NearestPeers(kb.ConvertKey(key), KValue)
routing.go:	peers := dht.routingTable.NearestPeers(kb.ConvertPeerID(id), KValue)
routing.go:	peers := dht.routingTable.NearestPeers(kb.ConvertPeerID(id), KValue)
util.go:var KValue = 10
```